languages/csharp/exercises/concept/constants/.docs/instructions.md

```diff
@@ -0,0 +1,21 @@
+The authentication system that you last saw in (TODO cross-ref-tba) is in need of some attention. You have been tasked with cleaning up the code.
+
+## 1. Set appropriate fields and properties to const
+
+This is a refactoring task. Add the `const` modifier to any members of `Authenticator` or `Identity` that you think appropriate.
+
+## 2. Set appropriate fields to readonly
+
+This is a refactoring task. Add the `readonly` modifier to any fields of the `Authenticator` class or the `Identity` struct that you think appropriate.
+
+## 3. Remove set accessor or make it private for any appropriate field
+
+Remove the `set` accessor or make it `private` for any appropriate property on the `Authenticator` class or `Identity` struct.
```

Comment:
In the example, I don't see this being applied to any properties. Am I mis-reading the code?

Reply:
You're right.  The students would not have responded well to a "trick" question.  Changed `GetAdmin()` to a property and changed task 3 to require immutability.  It wouldn't be a bad place to introduce the concept except that we already have 3 concepts.